.travis.yml

```diff
@@ -65,6 +65,7 @@ matrix:
         - cargo fmt --version || rustup component add rustfmt
         - cargo clippy --version || rustup component add clippy
         - cargo audit --version || cargo install cargo-audit
+        - cargo tarpaulin --version || cargo install cargo-tarpaulin
```

Comment:
This will only install `tarpaulin` for this job. I think you need to install it in Job `Tests on Linux`

Reply:
updated